### DISCUSSION TITLE
How to attach a middleware for static asset routes for css , js etc?

### DISCUSSION THREAD

**Discussion by Abrar535** [nuxt/nuxt#21531] [Questions (Nuxt 2)]
I have tried using server middleware to attach middleware to the routes exporting static files, But it doesnt work. For now i need to access log the hits to my nuxt server, i have done it for all the routes using serverMiddleware but the routes for static files are not working.

**Comment by mkierdev** [comment]
@danielroe @pi0 @manniL any chance it could be implemented? We stumbled across the same problem.

**Reply by mkierdev** [reply]
Now I saw that this question relates to Nuxt 2. I was pinging you about Nuxt 3..

**Reply by TheAlexLichter** [reply]
@mkierdev please refrain from pining team members like this! 

If you want to see this implemented, create an issue with reasoning and examples 👍
